unfoldingWord/translationCore
# Fixed Target translation not displayed in scripture pane or in Select area

#### This pull request addresses:

- Bug where text would not render when switching tools within the same project


#### How to test this pull request:

- Download this project 
[ar_eph_text_ulb.zip](https://github.com/unfoldingWord-dev/translationCore/files/1027801/ar_eph_text_ulb.zip)
- Delete the .apps folder
- Open the project and try switching tools within the same project. and make sure that the text in the scripture pane and Select area is rendered for both tools.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1671)
